.circleci/config.yml

```diff
@@ -314,27 +383,58 @@ jobs:
           name: Update Algolia Index
           command: |
             ALGOLIA_API_KEY=$ALGOLIA_PRIVATE_KEY bundle exec jekyll algolia --source jekyll --config jekyll/_config.yml
-  deploy:
+
+  deploy-preview:
```

Comment:
I moved the jobs around so that they match the order in which they are specified in `build_deploy` worflow